src/ocean/util/log/Config.d

```diff
@@ -447,8 +453,11 @@ public void configureLoggers ( Source = ConfigParser, FileLayout = LayoutDate,
         m_config = an instance of the MetaConfig class
         file_appender = delegate which returns appender instances to write to
                         a file
+        makeLayout      = A delegate that returns a `Layout` instance from
+                          a name, or throw on error.
+                          By default, wraps `ocean.util.log.Config.newLayout`
         use_insert_appender = true if the InsertConsole appender should be used
```

Comment:
I'd go with duplicating the doc (don't use `/// ditto`) and mention the reason for the two overloads in the doc for the second (i.e. move it out of the method body and into its "header" doc).